content/docs/command-reference/status.md

```diff
@@ -143,8 +142,8 @@ that.
   `.dvc` files to inspect. If there are no directories among the targets, this
   option has no effect.
 
-- `-r <name>`, `--remote <name>` - name of the [remote storage] to compare
-  against (see `dvc remote list`). Implies `--cloud`.
+- `-r <name>`, `--remote <name>` - name of the `dvc remote` to compare against
```

Comment:
I realize (tested) that this is working (i.e. implicit links using backquated commands) - but I'm wondering if it's a good a thing to intentionally rely on them when we already have explicit links, which is a more readable code imo (this rendering into a link is an obscure/implicit side affect).
explicit is better than implicit usually 😄, so even though it's cleaner to read - there's a major "unexpected" behavior

Reply:
> rendering into a link is an obscure/implicit side affect

I can see that criticism, and maybe you don't want to link absolutely every instance of every command name. On the other hand it saves a lot of time and no one has complained about too many links, that I know of.

Anyway, it would be a discussion for https://github.com/iterative/gatsby-theme-iterative 🙂

Reply:
yep. you're right, it's more of a question/discussion since it's already very prevalent